crates/ty_python_semantic/resources/mdtest/cycle.md

```diff
@@ -31,3 +31,16 @@ p = Point()
 reveal_type(p.x)  # revealed: Unknown | int
 reveal_type(p.y)  # revealed: Unknown | int
 ```
+
+## Too Many Cycles
```

Comment:
I think we need a better name for this section, and a `reveal_type` showing an `Unknown` result. Unless this results in a salsa cycle error if you do that?